packages/lesswrong/components/votes/OverallVoteAxis.tsx

```diff
@@ -0,0 +1,162 @@
+import { Components, registerComponent, getCollection } from '../../lib/vulcan-lib';
+import React from 'react';
+import { userIsAdmin } from '../../lib/vulcan-users/permissions';
+import moment from '../../lib/moment-timezone';
+import { useHover } from '../common/withHover';
+import { useCurrentUser } from '../common/withUser';
+import { forumTypeSetting } from '../../lib/instanceSettings';
+import { Comments } from '../../lib/collections/comments/collection';
+import { Posts } from '../../lib/collections/posts/collection';
+import { Revisions } from '../../lib/collections/revisions/collection';
+import classNames from 'classnames';
+import type { VotingProps } from './withVote';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  overallSection: {
+    display: 'inline-block',
+    height: 24,
+    paddingTop: 2
+  },
+  overallSectionBox: {
+    outline: `1px solid ${theme.palette.commentBorderGrey}`,
+    textAlign: 'center',
+    minWidth: 70
+  },
+  vote: {
+    fontSize: 25,
+    lineHeight: 0.6,
+    display: "inline-block"
+  },
+  voteScore: {
+    fontSize: '1.1rem',
+    marginLeft: 4,
+    marginRight: 4,
+    lineHeight: 1,
+  },
+  secondarySymbol: {
+    fontFamily: theme.typography.body1.fontFamily,
+  },
+  secondaryScore: {
+    fontSize: '1.1rem',
+    marginLeft: 2,
+    marginRight: 14
+  },
+  secondaryScoreNumber: {
+    marginLeft: 3,
+  },
+  tooltipHelp: {
+    fontSize: '1rem',
+    fontStyle: "italic"
+  }
+})
+
+const OverallVoteAxis = ({ document, hideKarma=false, voteProps, classes, showBox=false }: {
+  document: VoteableTypeClient,
+  hideKarma?: boolean,
+  voteProps: VotingProps<VoteableTypeClient>,
+  classes: ClassesType,
+  showBox: boolean
```

Comment:
```suggestion
  showBox?: boolean
```
This has a default value in the argument destructuring, but isn't marked as having an optional type. Changing the `:` to `?:` makes it properly optional.

Reply:
Good to know, thanks!